lib/ProductOpener/Config_off.pm

```diff
@@ -493,7 +493,7 @@ improvements
 
 # fields in product edit form, above ingredients and nutrition facts
 
-@product_fields = qw(quantity packaging brands categories labels origins manufacturing_places
+@product_fields = qw(quantity packaging brands 	brand_owner categories labels origins manufacturing_places
```

Comment:
This will make the brand owner field viewable and editable in the product edit form. I'm not sure if we really want to do that (as we will possibly get lots of strange values from users for other products).

Reply:
ok, fixing